<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newEventView</class>
 <widget class="QDialog" name="newEventView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Событие</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>События выбранной даты</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelSelDate">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="3">
    <widget class="QTableView" name="dateEventView"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxCase">
        <property name="text">
         <string>Дело</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboCases"/>
      </item>
      <item row="1" column="0" colspan="2" alignment="Qt::AlignTop">
       <widget class="QCheckBox" name="checkBoxUser">
        <property name="text">
         <string>Пользователь</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Владелец события:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Категория события</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="comboCatEvent"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Название события</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="editNameEvent"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Детали события</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Уведомить за</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QSpinBox" name="spinNumDays"/>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>дней</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="editDetailEvent"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="buttonSave">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="buttonChange">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="buttonCancel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="buttonDelEvent">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Удалить событие</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
